roles/cinder-common/templates/etc/cinder/cinder.conf

```diff
@@ -45,37 +45,45 @@ host = ceph
 glance_host = {{ endpoints.main }}
 glance_port = {{ endpoints.glance.port.backend_api }}
 
-{% if cinder.enabled_backends|length -%}
-enabled_backends = {{ cinder.enabled_backends|join(',') }}
+{% if ceph_pools.rbd_ssd.enabled and ceph_pools.rbd_hybrid.enabled -%}
```

Comment:
`ceph_pools` may not be defined at this point, there may need to be a default value set somewhere, because right now CI is failing due to this being undefined.